src/styles/card.scss

```diff
@@ -0,0 +1,126 @@
+.main {
+  --column-count: 1;
+
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: 46px 48px;
+  padding: 50px 40px;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .main {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .main {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .main {
+    --column-count: 4;
+  }
+}
+
+.card {
+  background-color: $white;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $grey;
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin: 40px 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $black;
+  }
+
+  &:hover {
+    transition: $transition;
+    transform: scale(1.2);
+    .card__name {
+      transition: $transition;
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $dark-grey;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    color: $black;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+    align-items: end;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+    margin-bottom: 16px;
+
+    &-name {
+      font-size: 12px;
+      line-height: 18px;
+      color: $dark-grey;
+    }
+
+    &-value {
+      font-weight: 700;
+      line-height: 18px;
+      color: $black;
+    }
+  }
+
+  &__button {
+    display: block;
+    margin: 0 auto;
+    text-transform: uppercase;
+    font-size: 14px;
+    color: $white;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+
+    border: transparent;
+    outline: none;
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+
+    background: $blue;
+
+    &:hover {
+      transition: $transition;
```

Comment:
Apply a transition to the class itself instead of `hover` so it will work both ways and specify to what properties transition should apply

Reply:
Fixed